RegressionTests/RegressionTests.swift

```diff
@@ -3,7 +3,7 @@ import SnapshotTesting
 import TestingExtensions
 import XCTest
 
-extension String: Error {}
+extension String: @retroactive Error {}
```

Comment:
We should probably just make an error type for this test instead of doing this. I foresee some issues continuing to rely on String here.